recipes/Time_Series/Bike_Sharing_Finetuning_with_Exogenous.ipynb

```diff
@@ -0,0 +1,1658 @@
PGaNKkSUpNTdUHH3yg4cOH2078wYMHy9vbWwsWLJB0afrYI488ooyMDB08eND2WrGxsfr000+1cuVK7dy5Uz169Kj8AwJQ4cgNAEaQHQDsRW4AMILsAGAE2VHzsfk6bHJycjRz5kzl5ubq1VdfVbt27SRJ/fr1U9euXbVy5UplZmZq6dKl+vvf/67jx4/L19dXVqtVJpNJQ4YMkZ+fn/7v//7PwUcCoLKQGwCMIDsA2IvcAGAE2QHACLKjdmDGCGzq1q0rNzc3Pfjgg2rXrp0KCwslSaNGjdKxY8dktVrl5eWlP/zhD+rdu7cmTJig2NhYmUwm/fzzz0pKStK4ceMcexAAKhW5AcAIsgOAvcgNAEaQHQCMIDtqB2aMoJSCggK5uLhIkiwWi8xms/74xz/Kw8ND7777ru1x8fHxGjJkiAoLC9WnTx/t3btX/v7+Wrt2rZo2beqo5gNwAHIDgBFkBwB7kRsAjCA7ABhBdtR8FEZwQ8HBwXrkkUc0ZcoUWSwWSZLZbFZ0dLT279+vffv2qWfPnpoyZYqDWwqgqiA3ABhBdgCwF7kBwAiyA4ARZEfNQmEEZTp9+rQGDBigzZs3KyAgQJKUn58vV1dXB7cMQFVFbgAwguwAYC9yA4ARZAcAI8iOmoc9RnBNJfWysLAweXp62k74BQsW6IknnlBSUpIjmwegCiI3ABhBdgCwF7kBwAiyA4ARZEfN5ezoBqBqMplMkqTw8HDde++92rZtmx599FHl5ORo1apV8vHxcXALAVQ15AYAI8gOAPYiNwAYQXYAMILsqLlYSgvXlZubqx49eujUqVNydXXVggUL9Mwzzzi6WQCqMHIDgBFkBwB7kRsAjCA7ABhBdtRMFEZQpjvuuEN+fn5atGiR6tSp4+jmAKgGyA0ARpAdAOxFbgAwguwAYATZUfNQGEGZioqK5OTk5OhmAKhGyA0ARpAdAOxFbgAwguwAYATZUfNQGAEAAAAAAAAAALWG2dENAAAAAAAAAAAAqCwURgAAAAAAAAAAQK1BYQQAAAAAAAAAANQaFEYAAAAAAAAAAECtQWEEAAAAAAAAAADUGhRGAAAAAAAAAABArUFhBAAAAAAAAAAA1BoURgAAAAA4zJAhQ/Tkk0+W++u2bdtWS5YsKffXBQAAAFD9URgBAAAAUCVQzAAAAABQGSiMAAAAAAAAAACAWoPCCAAAAIBKkZ2drcmTJ8vT01PNmzfXwoULbfcNGTJEsbGxmjNnjkwmk0wmk+2+sLAwDRw4UO7u7vL19dXs2bOVnZ1tuz8pKUljxoyRu7u72rVrpzVr1lz1dy9atEg9evSQh4eHfH19NXPmTGVlZdna5e3trQ0bNpR6zldffSUPDw9lZmaW91sBAAAAwIEojAAAAACoFE899ZR2796tjRs3auvWrdq1a5cOHDggSfriiy/UqlUrvfTSSzp79qzOnj0rSTp16pRGjhype++9VxEREVq3bp3CwsI0a9Ys2+s++OCDiouL086dO7VhwwYtX75cSUlJpf5us9mspUuX6ujRo/roo4+0Y8cOPf3005IkDw8P/f73v9fKlStLPWflypW677775OXlVZFvCwAAAIBKZrJarVZHNwIAAABAzZaVlaVGjRpp9erV+t3vfidJSk1NVatWrfToo49qyZIlatu2rZ588slSm7FPmzZNTk5Oeuedd2y3hYWFafDgwcrOztbPP/+szp07Kzw8XH379pUkRUVFqUuXLlq8ePF1N3bfsGGDZsyYoZSUFElSeHi4BgwYoLi4ODVv3lxJSUlq2bKltm/frsGDB1fMmwIAAADAIZgxAgAAAKDCnTp1Svn5+erXr5/ttoYNG6pz585lPu/w4cP68MMP5enpafsZMWKELBaLzpw5o2PHjsnZ2VkBAQG25/j7+6t+/fqlXmf79u0aNmyYWrZsKS8vLz3wwAM6f/68cnJyJEmBgYHq1q2bPvroI0nS6tWr1aZNGw0aNKic3gEAAAAAVQWFEQAAAABVVlZWlqZPn65Dhw7Zfg4fPqyTJ0+qQ4cON/UaMTExGj16tG655RZ9/vnn2r9/v95++21JUn5+vu1x06ZN04cffiipeBmtqVOnltrrBAAAAEDNQGEEAAAAQIXr0KGDXFxctG/fPtttFy5c0IkTJ2y/u7q6qqioqNTzevfurcjISHXs2PGqH1dXV/n7+6uwsFD79++3Pef48eNKS0uz/b5//35ZLBYtXLhQ/fv3V6dOnZSQkHBVGydNmqTY2FgtXbpUkZGRmjJlSjm+AwAAAACqCgojAAAAACqcp6enHn74YT311FPasWOHjhw5ogcffFBm86WvJG3bttW3336r+Ph4294fzzzzjPbu3atZs2bp0KFDOnnypDZu3GjbfL1z584aOXKkpk+frn379mn//v2aNm2a3N3dba/bsWNHFRQUaNmyZTp9+rRWrVqlFStWXNXGBg0aaPz48Xrqqac0fPhwtWrVqoLfFQAAAACOQGEEAAAAQKX4xz/+oYEDB2rMmDEKCQlRcHBwqb1BXnrpJcXExKhDhw5q0qSJJOmWW27R7t27deLECQ0cOFC33nqr5s2bpxYtWtiet3LlSrVo0UKDBw/W+PHj9eijj8rHx8d2f8+ePbVo0SK9/vrr6t69u9asWaNXX331mm18+OGHlZ+fr4ceeqiC3gUAAAAAjmayWq1WRzcCAAAAAKqCVatWac6cOUpISJCrq6ujmwMAAACgAjg7ugEAAAAA4Gg5OTk6e/asXnvtNU2fPp2iCAAAAFCDsZQWAAAAgFrvjTfekL+/v5o1a6bnnnvO0c0BAAAAUIFYSgsAAAAAAAAAANQazBgBAAAAAAAAAAC1BoURAAAAAAAAAABQa1AYAQAAAAAAAAAAtQaFEQAAAAAAAAAAUGtQGAEAAAAAAAAAALUGhREAAAAAAAAAAFBrUBgBAAAAAAAAAAC1BoURAAAAAAAAAABQa/x/CFQn2Evajp8AAAAASUVORK5CYII=",
```

Comment:
in general, it's better to put images in a seperate file,but I get how that can cause issues with making the notebook colab friendly

Reply:
These images are a result of plotting functions in the notebook.

Reply:
Ah I see that now. In other repos we enforce cleared outputs, not sure if thats M.O. here or not.